src/ui-components/QuickView.tsx

```
@@ -189,52 +194,78 @@
+									<span className='text-bodyBlue dark:text-blue-dark-high'>
+										{judgements
+											?.map(([, jud]) => jud.toString())
+											.join(', ')
+											?.split(',')?.[0] || 'None'}
+									</span>
+								</article>
+							)}
 						</div>
 					</div>
 				</div>
+				<div className={loginUserId && !isNaN(loginUserId) ? 'flex justify-between' : 'flex justify-start'}>
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)